library/include/G4CMPSolidUtils.hh

```diff
@@ -25,25 +25,43 @@
 #include "G4ThreeVector.hh"
 
 class G4VSolid;
+class G4VTouchable;
 
 class G4CMPSolidUtils {
   public:
     // Default constructor
-    G4CMPSolidUtils() : theSolid(0), verboseLevel(0) {;}
+    G4CMPSolidUtils() : theSolid(0), theTouchable(0), verboseLevel(0),
+                        verboseLabel("") {;}
```

Comment:
Do you want to make the default label "G4CMPSolidUtils" in this constructor, just for consistency?